### PR TITLE
cleanup2g simpler pr template

### PR DESCRIPTION
**Pull request by sdolenc**
porting https://github.com/Microsoft/edx-platform/pull/130 to edx-configuration

#### What does this PR do? Please provide some context

simplifying the pr template

#### Where should the reviewer start?

n/a. short review

#### How can this be manually tested? (brief repro steps and URL with change)

submit a pull request

#### What are the relevant TFS items? (list id numbers)

n/a

#### Definition of done:
- [x] Title of the pull request is clear and informative
- [x] Add pull request hyperlink to relevant TFS items
- [x] For large or complex change: schedule an in-person review session
- [x] This change has appropriate test coverage
- [x] Get at least two approvals

#### Reminders DURING merge
1. If you're merging from a feature branch into the development branch then "Squash and merge" to keep our history clean.
1. If merging from two longterm branches (like cherry picks from upstream, dev to release, etc) then "Create merge commit" to preserve individual commits.

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 2 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 3 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 4 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

**Review by manikarthikk — APPROVED** [review]

**Review by saravpal — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #### Where should the reviewer start?
 
-#### How can this be manually tested? (brief repro steps)
+#### How can this be manually tested? (brief repro steps and URL with change)
```
Comment (kevsers) [inline .github/PULL_REQUEST_TEMPLATE.md]: Although agree we should share the environment urls, I am not bought into the idea of sharing the dev urls publicly esp. for dev env. at this point. 
It seems there is actually a way to lock test/dev environments, but we have not done that yet. And not everybody is using internally acessible VMs at this point. Consider putting a reminder around that and if a public url is used, to share through emails.

Comment (kevsers) [reply .github/PULL_REQUEST_TEMPLATE.md]: That comment goes with all your PRs at this point - don't want to copy paste to all :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,18 +12,11 @@
 - [ ] Add pull request hyperlink to relevant TFS items
 - [ ] For large or complex change: schedule an in-person review session
 - [ ] This change has appropriate test coverage
+- [ ] Get at least two approvals
```
Comment (kevsers) [inline .github/PULL_REQUEST_TEMPLATE.md:15]: we can also add all comments/reviews addressed as a final chekpoint

Comment (kevsers) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: That comment goes with all your PRs at this point - don't want to copy paste to all :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sdolenc) [reply .github/PULL_REQUEST_TEMPLATE.md]: great point @kevsers and thanks for taking the time to chat with me about this in person. i'll update the phrasing for this and the other similar changes.

**Pull request merged**